src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -111,7 +113,9 @@ private static string[] GetDefaultLetterKeyALL(LetterKey letter)
                 LetterKey.VK_X => new string[] { "ẋ", "ξ" },
                 LetterKey.VK_Y => new string[] { "ÿ", "ŷ", "ý", "ẏ" },
                 LetterKey.VK_Z => new string[] { "ź", "ž", "ż", "ʒ", "ǯ", "ζ" },
-                LetterKey.VK_COMMA => new string[] { "¿", "¡", "∙", "₋", "⁻", "–", "≤", "≥", "≠", "≈", "≙", "±", "₊", "⁺" },
```

Comment:
According to @damienleroy, mapping to VK_PLUS, VK_MINUS is not going to work on French keyboards, therefore I'd suggest leaving it mapped to the VK_COMMA symbol as it is.

Reply:
Ok